bin/main.rb

```diff
@@ -0,0 +1,177 @@
+class Board
+  attr_accessor :cells
+  @cells = []
+
+  def initialize
+    @cells = [[" ", " ", " "],[" ", " ", " "],[" ", " ", " "]]
+  end
+
+  def display_board_game
+    puts "  |‾‾‾‾‾|‾‾‾‾‾|‾‾‾‾‾|"
+    puts "  |  #{@cells[0][0]}  |  #{@cells[0][1]}  |  #{@cells[0][2]}  |"
+    puts "  |7_____|8_____|9_____|"
+    puts "  |      |      |      |"
+    puts "  |  #{@cells[1][0]}  |  #{@cells[1][1]}  |  #{@cells[1][2]}  |"
+    puts "  |4_____|5_____|6_____|"
+    puts "  |      |      |      |"
+    puts "  |  #{@cells[2][0]}  |  #{@cells[2][1]}  |  #{@cells[2][2]}  |"
+    puts "  |1_____|2_____|3_____|"
+    puts ""
+  end  
+end
+
+# in this method we check if the input of the player is inside the board and we replace it with the symbol of each player
+def choose_move
+  case $chosen_move
+  when 1
+    if @cells[2][0].empty?
+      @cells[2][0] = "#{$current_player.symbol}" 
+    else 
+      return "Choose another cell"
+    end
+  when 2 
+    if @cells[2][1].empty?
+      @cells[2][1] = "#{$current_player.symbol}" 
+    else
+      return "Choose another cell"
+    end
+  when 3
+    if @cells[2][2].empty?
+      @cells[2][2] = "#{$current_player.symbol}" 
+      else 
+      return "Choose another cell"
+    end
+  when 4
+    if @cells[1][0].empty?
+      @cells[1][0] = "#{$current_player.symbol}" 
+    else 
+      return "Choose another cell"
+    end
+  when 5
+    if @cells[1][1].empty?
+      @cells[1][1] = "#{$current_player.symbol}" 
+    else 
+      return "Choose another cell"
+    end
+when 6
+if @cells[1][2].empty?
+  @cells[1][2] = "#{$current_player.symbol}" 
+else 
+  return "Choose another cell"
+end
+when 7
+if @cells[0][0].empty?
+  @cells[0][0] = "#{$current_player.symbol}" 
+else 
+  return "Choose another cell"
+end
+when 8
+if @cells[0][1].empty?
+  @cells[0][1] = "#{$current_player.symbol}" 
+else 
+  return "Choose another cell"
+end
+when 9
+if @cells[0][2].empty?
+  @cells[0][2] = "#{$current_player.symbol}" 
+else 
+  return "Choose another cell"
+end
+end
+end
+end
+
+# in the Game class we have the FirstSetup class working and getting the info from the players
+class Game
+  attr_accessor :active_board
+  def initialize
+    # @active_board = Board.new
+  end
+  
+  def players_info
+    puts "Welcome to our tic tac toe game!"
+    puts "Player 1 type your name, type enter and enter you symbol: "
+    $player_1 = FirstSetup.new(gets.chomp, gets.chomp)
+    puts "Welcome #{$player_1.name}, you are the first player, and you symbol is '#{$player_1.symbol}'"
+    puts "Player 2 type your name, type enter and type symbol: "
+    $player_2 = FirstSetup.new(gets.chomp, gets.chomp)
+    puts "Welcome #{$player_2.name}, you are the second player, and you symbol is '#{$player_2.symbol}'"
+    puts "Let's decide who is going first, let's flip a coin. press enter"
+    puts "#{$player_1.name}, you're HEADS. #{$player_2.name}, you're TAILS."
+    gets
```

Comment:
- [ ] [REQUIRED] The game breaks after the "coin toss". Please fix this.
Hint: The cause is most likely this `gets` statement on line 143.